app/models/game.rb

```diff
@@ -0,0 +1,10 @@
+class Game < ActiveRecord::Base
+
+  validates_presence_of :name, uniqueness: true
+  validates_presence_of :description
+  validates_presence_of :min_players
+  validates_presence_of :max_players
+  validates_presence_of :playing_time
+  validates_presence_of :complexity
```

Comment:
Use the newer validation format:

```ruby
validates :name, presence: true, uniqueness: true
```

This lets you combine multiple validations for a single attribute. Not sure if that `uniqueness: true` is doing anything because it's on a presence validator.